pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -95,6 +95,9 @@
         @Parameter(names = { "-q", "--receiver-queue-size" }, description = "Size of the receiver queue")
         public int receiverQueueSize = 1000;
 
+        @Parameter(names = { "--replicated" }, description = "Wether the subscription status should be replicated")
```

Comment:
"Wether" - > "Whether"

Reply:
Fixed. I think I'll never learn the correct spelling...